packages/docs/stories/components/radio-group.stories.ts

```diff
@@ -131,7 +137,7 @@ export const Invalid: Story = {
     }
   },
   render: () => html`
-  <form class="custom-validity">
+  <form>
```

Comment:
I thought it would better readable to change the styling from 

![image](https://github.com/synergy-design-system/synergy-design-system/assets/115139104/3b44377c-f283-4193-886a-cd116adc32a0)

to 

![image](https://github.com/synergy-design-system/synergy-design-system/assets/115139104/2f7072d9-78ea-4ef5-812a-6d089822cd4f)

In the end it has the same effect (same margins). 
But I can revert it, if you think it´s better the other way